templates/cohorts/new_cohort.html

```diff
@@ -195,6 +195,24 @@ <h4 class="panel-title pull-left">Selected Filters</h4>
             </div>
         </div>
 
+        <div class="cohort-info panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">Details</h4>
+            </div>
+            <div class="panel-body">
```

Comment:
Done, and, I added in a space class to pad the rows; apparently this is available in Bootstrap 4 but in the version we have oyu need to space the rows vertically with your own classes.